src/software/jetson_nano/display/screens/chip_and_kick_screen.py

```diff
@@ -0,0 +1,142 @@
+from screen import Screen
+
+CHIP = 0
+KICK = 1
+
+PADDING = 6
+BASE_Y = 20
+
+"""
+This screen is used to edit the chip and kick speed settings
+"""
+
+
+class ChipAndKickScreen(Screen):
+    def __init__(self, lcd_display, redis_dict, status_codes):
+        self.enable = False if redis_dict["chip and kick enable"] == 0 else True
+        self.speeds = [
+            redis_dict["chip speed"],
+            redis_dict["kick speed"],
+        ]
+
+        # Defining this screens actions
+        def menu():
+            """ Go to the menu screen """
+            self.curr_action = 0
+
+            return {self.status_codes["change screen"]: "Menu"}
+
+        def set_chip_and_kick_speed():
+            """ Enable and disable settings """
+            if self.enable:
+                self.enable = False
+            else:
+                self.enable = True
+
+            self.update_screen()
+            return {
+                self.status_codes["update redis"]: {
+                    "redis key": "chip and kick enable",
+                    "value": 1 if self.enable else 0,
+                }
+            }
+
+        def set_chip():
+            """ Set chip speed """
+            return {
+                self.status_codes["edit"]: {
+                    "param": self.speeds,
+                    "setting": CHIP,
+                    "delta": 0.5,
+                    "redis key": "chip speed",
+                }
+            }
+
+        def set_kick():
+            """ Set kick speed """
+            return {
+                self.status_codes["edit"]: {
+                    "param": self.speeds,
+                    "setting": KICK,
+                    "delta": 0.5,
+                    "redis key": "kick speed",
+                }
+            }
+
+        # Listing actions for Config Wheels Screen
+        self.actions = ["Set Chip and Kick Speed", "Chip", "Kick", "Menu"]
+        self.action_map = {
+            self.actions[0]: set_chip_and_kick_speed,
+            self.actions[1]: set_chip,
+            self.actions[2]: set_kick,
+            self.actions[3]: menu,
+        }
+
+        def draw_screen():
+            """ Wheels Screen Layout """
+            self.lcd_display.prepare()
+
+            cursor = ">"
+            cursor_size = self.font.getsize(cursor)[0]
+            cursor_pos_x = 0
+            if self.curr_action != len(self.actions) - 1:
+                cursor_pos_y = 20 + self.font_size * self.curr_action
```

Comment:
I think I saw 20 earlier in this PR, is it a constant that should be pulled out?

Reply:
added a `constants.py` file which holds the constants used.